packages/dataviews/src/types.ts

```diff
@@ -389,6 +396,11 @@ export interface ViewGrid extends ViewBase {
 		 * The fields to use as badge fields.
 		 */
 		badgeFields?: string[];
+
+		/**
+		 * The number of grid columns.
+		 */
+		gridColumns?: number;
```

Comment:
Why does the UI say "preview size" and the setting say "gridColumns". This feels like we're probably not storing the right thing here.

Reply:
What does it store today? What values?

Reply:
I already updated to `previewSize`. It stores number values (0, 1, 2, etc..).